run_coverage_batches.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Find all test files
+find test -name "*_test.dart" > all_tests.txt
+split -l 50 all_tests.txt batch_
+
+mkdir -p coverage_parts
+count=0
+for batch in batch_*; do
+  count=$((count+1))
+  echo "Running batch $count..."
+  flutter test $(cat $batch) --coverage
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)